### PR TITLE
macros tutorial_plugins

### PR DESCRIPTION
**Pull request by KetpuntoG**
#739

### CODE REVIEW

File: demonstrations/tutorial_plugins_hybrid.py [hunk 1]
```diff
@@ -66,7 +66,7 @@
 
 4. **Finally, we measure the mean photon number** :math:`\left\langle \hat{n}\right\rangle` of the second wire, where
 
-   .. math:: \hat{n} = \ad\a
+   .. math:: \hat{n} = \hat{a}^{\dagger}\hat{a}
```
Comment (isaacdevlugt) [inline demonstrations/tutorial_plugins_hybrid.py:69]: 🎩 🗡️

**Review by isaacdevlugt — APPROVED** [review]
LGTM!

File: demonstrations/tutorial_plugins_hybrid.py (hunk 1 above)
Comment (CatalinaAlbornoz) [reply demonstrations/tutorial_plugins_hybrid.py:69]: It looks fine in Safari but it looks weird in Chrome for me. I seem to be the only one having weird rendering issues in Chrome so if nobody else sees this then you can approve this. However if this isn't just me then we should fix it.

image

Comment (KetpuntoG) [reply demonstrations/tutorial_plugins_hybrid.py:69]: Oh you are right, in chrome is weird. Good catch, I'll ask in the channel, thanks!

**Review by CatalinaAlbornoz — APPROVED** [review]

**Pull request merged**